Unreal/Plugins/AirSim/Source/Vehicles/CustomCar/CustomCarPawn.cpp

```diff
@@ -205,6 +222,25 @@ void ACustomCarPawn::Tick(float Delta)
     this->SetActorLocationAndRotation(newPose.location, newPose.rotation);
 
     pawn_events_.getPawnTickSignal().emit(Delta);
+
+    if (even) { // Set tire angle
+        FRotator rot(0,tire_angle_,0);
+        wheel_fl_->SetRelativeRotation(rot, false, NULL, ETeleportType::None);
+        wheel_fr_->SetRelativeRotation(rot, false, NULL, ETeleportType::None);
+        even = false;
+    }
+    else { // Set tire speed
+        rotating_movement_fl_->RotationRate.Roll = vehicle_state_.fl_wheel_state.angular_velocity*RAD2DEG;
+        rotating_movement_fr_->RotationRate.Roll = vehicle_state_.fr_wheel_state.angular_velocity*RAD2DEG;
+        rotating_movement_rl_->RotationRate.Roll = vehicle_state_.rl_wheel_state.angular_velocity*RAD2DEG;
+        rotating_movement_rr_->RotationRate.Roll = vehicle_state_.rr_wheel_state.angular_velocity*RAD2DEG;
+
+        rotating_movement_fl_->SetUpdatedComponent(wheel_fl_);
+        rotating_movement_fr_->SetUpdatedComponent(wheel_fr_);
+        rotating_movement_rl_->SetUpdatedComponent(wheel_rl_);
+        rotating_movement_rr_->SetUpdatedComponent(wheel_rr_);
```

Comment:
so, the docs say that the movement component automatically registers itself with a tick and gets executed automatically. Can we update the component with the new speed outside of the tick function? That would solve the issue.

Reply:
This still causes it to run in the same tick as the tire yaw setting code, I tried and it doesn't work :/